pkg/network/controllers/vm.go

```diff
@@ -154,3 +123,27 @@ func (v *VMController) vmiInterfacesPatch(newVmiSpec *v1.VirtualMachineInstanceS
 
 	return err
 }
+
+func hasOrdinalSecondaryIfaces(
```

Comment:
Can we add this to https://github.com/kubevirt/kubevirt/blob/06eae09ea43fe7b3ea0ef258c592dc512cf65b55/pkg/network/namescheme/networknamescheme.go ?

Reply:
Done.